### ISSUE TITLE
issue related to corner_line change name

### ISSUE THREAD

**Issue by agrouaze** [umr-lops/xsar_slc#44]
```python
File "/home1/datahome/agrouaze/sources/git/xsar_slc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 138, in <module>
    generate_IW_L1Bxspec_product(slc_iw_path=slc_iw_path,output_filename=output_filename, dev=args.dev,
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/scripts/L1B_xspectra_IW_SLC_IFR.py", line 75, in generate_IW_L1Bxspec_product
    one_subswath_xspectrum_dt = proc.compute_subswath_xspectra(dt,pol=polarization.upper(),
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/xsarslc/processing/xspectra.py", line 41, in compute_subswath_xspectra
    inter_xs = compute_IW_subswath_interburst_xspectra(dt, **kwargs)
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/xsarslc/processing/xspectra.py", line 198, in compute_IW_subswath_interburst_xspectra
    interburst_xspectra = tile_bursts_overlap_to_xspectra(burst0, burst1, dt['geolocation_annotation'], dt['calibration'], tile_width,
  File "/home1/datahome/agrouaze/sources/git/xsar_slc/xsarslc/processing/interburst.py", line 139, in tile_bursts_overlap_to_xspectra
    corner_lons = FullResolutionInterpolation(corner_line, corner_sample, 'longitude', geolocation_annotation,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/dataarray.py", line 2824, in drop
    ds = self._to_temp_dataset().drop(labels, dim, errors=errors, **labels_kwargs)
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/dataset.py", line 5171, in drop
    return self.drop_vars(labels, errors=errors)
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/dataset.py", line 5051, in drop_vars
    self._assert_all_in_dataset(names)
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xarray/core/dataset.py", line 5018, in _assert_all_in_dataset
    raise ValueError(
ValueError: One or more of the specified variables cannot be found in this dataset
```

**Comment by lanougue** [comment]
I'll give a look next week!

**Comment by lanougue** [comment]
solved in #45
